tests/test_utils.py

```diff
@@ -48,6 +49,77 @@ def test_list_exchanges(capsys):
     assert re.search(r"^bittrex$", captured.out, re.MULTILINE)
 
 
+def test_list_timeframes(capsys):
+
+    args = [
+        "list-timeframes",
+    ]
+    pargs = get_args(args)
+    pargs['config'] = None
+    with pytest.raises(OperationalException,
+                       match=r"This command requires a configured exchange.*"):
+        start_list_timeframes(pargs)
+
+    # Test with --config config.json.example
+    args = [
+        '--config', 'config.json.example',
+        "list-timeframes",
+    ]
+    start_list_timeframes(get_args(args))
+    captured = capsys.readouterr()
+    assert re.match("Timeframes available for the exchange `bittrex`: "
```

Comment:
@xmatthias do we have a `out_has` function which behaves like `log_has` but on the output printed? As you see, strings here are not regexps to use re.match against them...

Reply:
no we don't - but if you check the first 2 functions in `conftest.py` - writing that is not difficult...